### PR TITLE
Add Metadata for airports_graph.py and geojson_source.py

### PR DESCRIPTION
**Pull request by dasha-moskv**
As the title implies, I added meta data for \examples\topics\graph\airports_graph.py and examples\topics\geo\geojson_source.py.
- [x] issues: ref #11765

### CODE REVIEW

File: examples/topics/graph/airports_graph.py [hunk 1]
```diff
@@ -1,3 +1,15 @@
+'''
+This example creates an interactive map visualization of airport routes and locations using Bokeh.
+
+.. bokeh-example-metadata::
+    :sampledata: airport_routes, us_states
+    :apis: bokeh.models.ColumnDataSource, bokeh.models.StaticLayoutProvider,
+           bokeh.models.NodesAndLinkedEdges, bokeh.models.TapTool, bokeh.plotting.figure,
+           bokeh.plotting.show
```
Comment (bryevdv) [inline examples/topics/graph/airports_graph.py]: Unfortunately our homegrown sphinx extension is not smart enough to handle multiple lines, yet, so every thing will need to go on a single line. We have a fairly large column limit (160 I think) but you still may want to remove some of these for compactnness

File: examples/topics/graph/airports_graph.py [hunk 2]
```diff
@@ -1,3 +1,15 @@
+'''
+This example creates an interactive map visualization of airport routes and locations using Bokeh.
```
Comment (bryevdv) [inline examples/topics/graph/airports_graph.py]: Start docstrings on the same line with a leading space, for consistency with existing examples (same comment in the other file in this PR as well)

```suggestion
''' This example creates an interactive map visualization of airport routes and locations using Bokeh.
```

**Review by bryevdv — COMMENTED** [review]
Thanks for the PR @dasha-moskv just a couple of minor comments

**Comment by bryevdv** [comment]
almost there @dasha-moskv just some trailing space tripping up the linter
```
E         examples/topics/graph/airports_graph.py:5:132: W291 [*] Trailing whitespace
```

**Review by bryevdv — APPROVED** [review]

**Comment by bryevdv** [comment]
Thanks for the PR @dasha-moskv !

**Comment by dasha-moskv** [comment]
Thank you!

**Pull request merged**